README.md

```diff
@@ -17,6 +18,24 @@ This library provides the ability to process messages passed onto a dataLayer qu
 - [License](#license)
   
 
+## Quick Start
+First, install the package with your favorite package manager
+```bash
+npm install data-layer-helper --save-dev
+// or
+yarn add data-layer-helper --dev
+```
+
+Next, require the code in your javascript
```

Comment:
Nit: I would change the wording to import not require as require() is a different API than the example below.